slam-libraries/viam-cartographer/src/io/submap_painter.cc

```diff
@@ -113,5 +113,27 @@ cartographer::io::PaintSubmapSlicesResult PaintSubmapSlices(
                                                      origin);
 }
 
+// Draw a position on the current map
+void DrawPositionOnSurface(
```

Comment:
added a new function here to mark the pose on top of the final map, that way we do not run into issues with masking & positions becoming faded